code/__DEFINES/mobs.dm

```diff
@@ -448,6 +448,7 @@
 #define FLASH_PROTECTION_NONE 0
 #define FLASH_PROTECTION_FLASH 1
 #define FLASH_PROTECTION_WELDER 2
+#define FLASH_PROTECTION_MAXIMUM 3
```

Comment:
Ostensibly this is a lie, because maximum flash protection would be `INFINITY`. 

Also, why go out of your way to only protect `SENSITIVE` and not `HYPER_SENSITIVE`?

Reply:
Good point